scripts/in_container/run_prepare_airflow_packages.sh

```diff
@@ -55,9 +55,10 @@ function prepare_airflow_packages() {
         else
             if [[ ${AIRFLOW_VERSION} != *${VERSION_SUFFIX_FOR_PYPI} ]]; then
                 echo
-                echo "${COLOR_RED}The requested PyPI suffix ${VERSION_SUFFIX_FOR_PYPI} does not match the one in ${AIRFLOW_VERSION}. Exiting.${COLOR_RESET}"
+                echo "${COLOR_YELLOW}The requested PyPI suffix ${VERSION_SUFFIX_FOR_PYPI} does not match the one in ${AIRFLOW_VERSION}. Overriding it with one from ${AIRFLOW_VERSION}.${COLOR_RESET}"
                 echo
-                exit 1
+                VERSION_SUFFIX_FOR_PYPI=$(printf "%s\n" "${AIRFLOW_VERSION##*.}")
```

Comment:
Wouldn't this set the suffix to `0b1`? (`2.4.0b1` -> `0b1`)

Reply:
Ah right. I thought we are adding "." after number and tested it with it. Moment

Reply:
BTW., I wish I had completed python conversion even for the in-container scripts. I learned to hate bash while doing so :D

Reply:
Why do we even need to specify this version suffix when building the wheels for tests?